### ISSUE TITLE
Error building oifitslib

### ISSUE THREAD

**Issue by stellaCL** [jsy1001/oifitslib#5]
Hi,

I have an issue while building the library. I get error messages that it cannot find cfitsio and glib. However, these are already installed with conda. Is there a way to change the path, so that cmake can find them?

OS: Ubuntu 20.04

Thanks!

**Comment by jsy1001** [comment]
Hi @stellaCL The cmake files for oifitslib use the `pkg-config` utility to locate cfitsio and glib. If your installation of these libraries contains .pc files it should be sufficient to set the `PKG_CONFIG_PATH` environment variable to the location(s) of the .pc files. Please let me know if you can't get this approach to work.

**Comment by stellaCL** [comment]
Hi @jsy1001  Thank you for the help! It all worked in the end. I also figured out that I had skipped the installation of SWIG and Doxygen, so I went back to repeat the whole process.

Just some feedback from me since it's a new installation and in case another Anaconda user has the same issues:

1) The first problem I encountered was a cmake error: `"CMAKE_CXX_COMPILER-NOTFOUND"` . This was fixed once I run the following `sudo apt-get update && sudo apt-get install build-essential`
2) The next step was to install `pkg-config`, `swig`, and `doxygen` with conda (the last is in the conda-forge channel). I already had `cfitsio` and `glib` installed via conda.
3) Once all this is done, then installation run smoothly, and I can see the API documentation as well. I also run a test with `oifits-upgrade` and `oifits-check` and it's all good.
